src/mach/load_command.rs

```diff
@@ -1596,6 +1672,10 @@ impl CommandVariant {
             DylibCodeSignDrs(comm) => comm.cmdsize,
             LinkerOption(comm) => comm.cmdsize,
             LinkerOptimizationHint(comm) => comm.cmdsize,
+            VersionMinTvos(comm) => comm.cmdsize,
+            VersionMinWatchos(comm) => comm.cmdsize,
+            DyldExportsTrie(comm) => comm.cmdsize,
+            DyldChainedFixups(comm) => comm.cmdsize,
```

Comment:
We don’t need to do this now but might be a good idea to add non exhaustive annotations to these enums as much well